lennykamande/stackoverflow.github.io
# Ft get all questions 159850844

## What does this PR do?
Gets all questions asked by all users in the system.
### Description of Task to be completed?
Have the following UI available.
  * gets all questions asked and returns in json formart

### How should this be manually tested?
Make sure that you have installed Postman or ARC

* Clone the repo on to your local machine
* Cd into the folder 
* Run Python run.py 
* Open up Postman 
* Make a get request to http://localhost/5000/api/v1/questions

### Any background context you want to provide?
There are user interfaces for the stackoverflow Lite that have not been developed yet.
### What are the relevant pivotal tracker stories?
#159850844
